kratos/utilities/coordinate_transformation_utilities.h

```diff
@@ -119,14 +119,14 @@ class CoordinateTransformationUtils {
 	 @param rLocalVector Local RHS vector
 	 @param rGeometry A reference to the element's (or condition's) geometry
 	 */
-	void Rotate(TLocalMatrixType& rLocalMatrix,
+	virtual void Rotate(TLocalMatrixType& rLocalMatrix,
```

Comment:
@jcotela i let you comment on this (it looks ok to me, just want to be sure it still works with the incompressible fluid)

Reply:
@RiccardoRossi I implemented these changes, I would expect someone else to review it ;)

Just for the record, I don't expect differences for the incompressible version, I only changed a few things to allow the derivation.

Reply:
changes look correct to me.